data.sql

```diff
@@ -1,5 +1,6 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals
+VALUES (DEFAULT, 'Agumon', '2020-2-3', 0, TRUE, 10.23),
```

Comment:
- [ ] inserting into a table should look something like below:
```
INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
   VALUES ('Agumon', '2020/2/3', '0', 'true', '10.23');
```

Reply:
@mikethreels Thanks for the review. I didn't know it was mandatory to do it that way. I'll make sure to do it the same way you do.